.github/workflows/webpack.yml

```diff
@@ -16,10 +14,10 @@ jobs:
       steps:
         - uses: actions/checkout@v4
     
-        - name: Use Node.js ${{ matrix.node-version }}
+        - name: Use Node.js 
           uses: actions/setup-node@v3
           with:
-            node-version: ${{ matrix.node-version }}
+            node-version: "16.x"
```

Comment:
is 16 the best we can do with this action? aren't we developing on v20?

Reply:
good point! i've been using 19, so i updated this to be 19 or higher